themes/tiddlywiki/vanilla/base.tid

```diff
@@ -333,16 +333,17 @@ button svg, button img {
 }
 
 .tc-btn-boxed {
-	font-size: 0.6em;
```

Comment:
Ah, now I see. I am now using the actual button styles in the Tools panels both tiddler and sidebar... and doing that I saw that the "icon + text" variant didn't work out at all... and that it would only be ok if the buttons had no style at all, which seemed wrong as well, because after all, there are button styles... and so they should apply consistently.